src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -92,6 +92,9 @@
 
          Available options:
          "reboot" (type 'b')
+         "install-packages" (type 'as')
+         "uninstall-packages" (type 'as')
+         "install-local-packages" (type 'ah')
```

Comment:
Makes sense to me!  Yeah, I think `CreateDeployment` threw me at first into thinking it was from-scratch.  `UpdateDeployment` is cool by me.